src/components/Missions/Missions.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMissionsAsync } from '../../redux/missions/missionsSlice';
+import MissionRow from './MissionRow'; // Make sure to import your MissionRow component
+import './missions.css';
+
+function Missions() {
+  const missions = useSelector((state) => state.missions.missionData);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!missions.length) {
+      dispatch(fetchMissionsAsync());
+    }
+  }, [dispatch, missions.length]);
```

Comment:
- [ ] All Redux actions should be dispatched once, I can see that you dispatched `missions` actions twice here and inside the `MissionContainer.js` component lines 13 to 17. Please fix that.

![image](https://github.com/BHObserver/Space-Travelers-Hub/assets/68796232/62ceba1c-5efe-41ba-ad26-3f76357b6779)

- [ ] As Per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#:~:text=React%20and%20Redux.-,Set%20up%20testing%20libraries(React%20Testing%20Library%20and%20Jest).,-Do%20not%20preserve), Please write unit tests with React Testing Library and Jest.